cylc/flow/cfgspec/globalcfg.py

```diff
@@ -643,19 +643,42 @@
 
             .. versionadded:: 8.0.0
         '''):
-            Conf('plugins', VDR.V_STRING_LIST,
-                 ['health check', 'reset bad hosts'],
-                 desc='''
-                     Configure the default main loop plugins to use when
-                     starting new workflows.
+            Conf(
+                'plugins',
+                VDR.V_STRING_LIST,
+                ['health check', 'reset bad hosts'],
+                desc='''
+                    Configure the default main loop plugins to use when
+                    starting new workflows.
+
+                    Only enabled plugins are loaded, plugins can be enabled
+                    in three ways:
```

Comment:
By my count, that's two ways!

Reply:
Derp, I forgot that enabling per-workflow wasn't an option.

```suggestion
                    Only enabled plugins are loaded, plugins can be enabled
                    in two ways:
```